apps/hes/fsw/platform_inc/hes_platform_cfg.h

```diff
@@ -109,8 +109,13 @@ extern "C" {
 */
 #define HES_STARTUP_TIMEOUT_MSEC    (1000)
 
-/* TODO:  Add Doxygen markup. */
-#define HES_CUSTOM_DEV_PATH       "/tmp/ttyS0"
+
```

Comment:
See my comment above.  It applies similarly here.  Just remove the "TO_CPD" because the reference build might not have that defined.